src/examples/button/SplitButtonDanger.tsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState } from 'react';
+import { Row, Col } from '@zendeskgarden/react-grid';
+import { SplitButton, Button, ChevronButton } from '@zendeskgarden/react-buttons';
+import { Dropdown, Trigger, Menu, Item } from '@zendeskgarden/react-dropdowns';
+
+const Example = () => {
+  const [rotated, setRotated] = useState<boolean | undefined>();
```

Comment:
What do you think about typing this out as only a `boolean`? It seems like initializing `useState` with no value (e.g. `undefined`) the TypeScript compiler use type inference to allow `undefined` values.

Reply:
I followed an existing pattern for this convention. Git blame is telling me @jzempel set this up. I don't think I'm qualified to weigh in on this particular one.

Reply:
...which would've been a mindless copy/paste motion on my part. Let's go with Henry's instinct on this one and clean up the code, if possible.